airflow/serialization/serialization_base.py

```diff
@@ -43,8 +44,8 @@
 FAILED = 'serialization_failed'
 
 
-class Serialization:
-    """Serialization provides utils for serialization."""
+class SerializationBase:
```

Comment:
Maybe `BaseSerialization` to match the convention with `BaseOperator` ?

Reply:
Will do!

Reply:
Done.